HACKING.md

```diff
@@ -100,3 +100,43 @@ To change the project's formatting, linting and test dependencies:
    ```
 
 3. Commit everything to git and send a pull request
+
+Testing Manually
+----------------
+
+To test it manually you can install your local development copy of
+`tox-recreate` into the local development environment of another tox-using
+project such as
+[cookiecutter-pypackage-test](https://github.com/hypothesis/cookiecutter-pypackage-test):
+
+1. Install a local development copy of `cookiecutter-pypackage-test` in a temporary directory:
+
+   ```terminal
+   git clone https://github.com/hypothesis/cookiecutter-pypackage-test.git /tmp/cookiecutter-pypackage-test
+   ```
+
+2. Run `cookiecutter-pypackage-test`'s `make sure` command to make sure that
+   everything is working and to trigger tox to create its `.tox/.tox`
+   venv:
+
+   ```terminal
+   make --directory "/tmp/cookiecutter-pypackage-test" sure
```

Comment:
Is `make sure` missing a `sure: python` (the python bit) 

I had to call `bin/make_python`

Reply:
Yes, that looks like a bug in the cookiecutter. Opened an issue: https://github.com/hypothesis/cookiecutters/issues/48